include/seqan3/alphabet/union_alphabet.hpp

```diff
@@ -258,11 +308,36 @@ struct union_alphabet
         return value >= rhs.value;
     }
     //!@}
+
+protected:
+    //! \privatesection
+    // conversion tables
+
+    constexpr auto from_base_(const variant_type & alphabet_v) const {
```

Comment:
This function is dependent on the `variant_type` definition, whereas the 
other functions are independent of the class.

And they are in the namespace, s.t. I can test them from the outside.